library/core/src/num/nonzero.rs

```diff
@@ -80,10 +103,8 @@ impl_zeroable_primitive!(isize);
 /// ```
 #[unstable(feature = "generic_nonzero", issue = "120257")]
 #[repr(transparent)]
-#[rustc_layout_scalar_valid_range_start(1)]
-#[rustc_nonnull_optimization_guaranteed]
```

Comment:
This attr is responsible for the FFI safety lints and needs to stay

Reply:
Okay, I removed this because otherwise the following line unwraps a `None`:

https://github.com/rust-lang/rust/blob/2e3581bca93fdcce474e17cd43430b594a7955a0/compiler/rustc_lint/src/types.rs#L1105